internal/lpastore/resolving_service.go

```diff
@@ -37,7 +37,7 @@ func (s *ResolvingService) Get(ctx context.Context) (*Lpa, error) {
 
 	lpa, err := s.client.Lpa(ctx, donor.LpaUID)
 	if errors.Is(err, ErrNotFound) {
-		lpa = &Lpa{}
+		lpa = donorProvidedDetailsToLpa(donor)
```

Comment:
I wanted us to not do this, but we need to so that we can show data when the certificate provider has got the first email but the donor hasn't signed